template.json

```diff
@@ -21,22 +21,22 @@
             "stats": {
                 "might": {
                     "pool": {
-                        "current": 0,
-                        "maximum": 0
+                        "value": 0,
```

Comment:
Be wary when changing template.json! Adding new fields is OK since Foundry will simply add them to instances, deleting them is fine too if you're fine wtih losing the data, but renaming fields WILL destroy the former data. In this case, adding a migration would be necessary - and simple, too, since you're simply renaming the properties.

Reply:
Yeah, I figured this change would probably cause issues. But this seemed to be the only way to get the pools to properly show up as bars, so it seemed like we'd need to tackle it at some point anyway.

Reply:
I figured that was the reason. Don't worry, as I said, renaming a property is fairly trivial, I'll handle it, no problem.